src/components/Discussion/Tree/BoardComment.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { css } from 'glamor'
+import CommentNode, { CommentProps } from './CommentNode'
+import { CommentEmbed } from '../Internal/Comment'
+import { mUp } from '../../../theme/mediaQueries'
+
+const styles = {
```

Comment:
the grid layout here was whack because it created a new grid for every comment and did not align across multiple comments.